cli/remote_build.go

```diff
@@ -622,6 +622,8 @@ func defineBranch(r *git.Repository, hintBranch string) (string, error) {
 func (p *RemoteCmd) fetchProject(urls []string) (*Project, GitRemote, error) {
 	var empty GitRemote
 	v := url.Values{}
+	urlsList := fmt.Sprintf("URL's used to do the search: %s", urls)
```

Comment:
Maybe just print this out to avoid multi-line errors? Like "Couldn't find any project using the following URLs...." (also there's no `'` in the plural or URLs)